.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  release:
+    types:
+    - created
+name: Build Release
+jobs:
+  release-assests:
+    name: release kubectl-karmada
+    runs-on: ubuntu-18.04    
+    strategy:
+      matrix:
+        include:
+          - goarch: amd64
+            goos: linux
+          - goarch: amd64
+            goos: darwin
+          - goarch: arm64
+            goos: linux
+          - goarch: arm64
+            goos: darwin
+    steps:
+    - uses: actions/checkout@master

```

Comment:
Should we check out the code from a tag?
Like:
```
- uses: actions/checkout@master
   with:
     ref: refs/tags/v0.9.0
```

Reply:
`ref: ''`：
> The branch, tag or SHA to checkout. When checking out the repository that
> triggered a workflow, this defaults to the reference or SHA for that event.
> Otherwise, uses the default branch.
   
I think that is what we want.